Beheerlandschap.md

```diff
@@ -23,36 +25,17 @@ De relatie tussen deze omgevingen is gevisualiseerd in de onderstaande image:
-LET OP! API Beheert slechts de configuratie van de testscripts van de API standaarden die zij beheert. API Beheer beheert dus niet de API testvoorziening zelf.
-
-* developer.overheid.nl is een portaal waar ontwikkelaars informatie kunnen vinden over alle API's die in overheidsland gebruikt worden. Beheerders van overheids API's melden zelf hun eigen API's aan en beheren die aanmeldingen ook zelf.
-
-* Aangezien GEMMA Online voor velen de ingang is naar meer informatie over de VNG Realisatie standaarden is er in die omgeving ook voorzien in informatie over de API standaarden.
-Dit is echter tot een minimum beperkt, vanuit GEMMA Online wordt slechts globale informatie verstrekt over deze standaarden en voor meer informatie wordt daar verwezen naar de ref.tst.vng.cloud omgeving.
-
-Tenslotte speelt ook de softwarecatalogus (SWC) binnen het API Beheerlandschap een rol. De SWC is een product van en voor gemeenten die helpt het applicatielandschap in beeld te brengen. De API's maken daar ook deel van uit en moeten dus in de SWC binnen dit landschap geconfigureerd kunnen worden. Beheer van de SWC is geen verantwoordelijkheid van API Beheer maar API Beheer zal de beheerders van de SWC wel van informatie moeten voorzien zodat zij de configuratie kunnen regelen. 
-
-Daarnaast kan het zijn dat, zoals met het StUF TestPlatform, de resultaten van de testen op de API TestVoorziening moeten worden ingelezen in de SWC. Die relatie is echter nog niet helder.
-
-
-
-
-
-
-
-
+Het daadwerkelijk vervaardigen van de API standaarden en coderen van de Referentie Implementaties gebeurt in de GitHub repositories van de API's. De andere omgevingen spelen een andere rol. Hieronder beschrijven we die rol en de relatie tussen die omgevingen.
+* In de ref.tst.vng.cloud omgeving wordt algemene informatie m.b.t. de Zaakgericht werken API's ter beschikking gesteld zoals de context waarbinnen en de wijze waarop de API's zijn ontwikkeld. Daarnaast bevat het echter ook documentatie over de inhoudelijke/technische kant van de API's. Zo bevat 
+https://ref.tst.vng.cloud/zrc/api/v1/schema/ bijvoorbeeld inhoudelijk/technische informatie over de zaakregistratiecomponent API en https://ref.tst.vng.cloud/drc/api/v1/schema/ over de documentregistratiecomponent API.<br/><br/>De algemene informatie wordt direct vanuit GitHub m.b.v. scripts in het tool Hugo gegenereerd (pijl 1 in het bovenstaande diagram). 
+De inhoudelijk/technische documentatie wordt vanuit de docker images in hub.docker.com/u/vngr gegenereerd (pijl 10). Daarmee wordt gewaarborgd dat de documentatie in ref.tst.vng.cloud altijd in sync loopt met de laatste versie van de gepubliceerde docker bestanden. De bron daarvoor isn echter ook de GitHub omgeving wat betekent dat de GitHub omgeving in feite de content management omgeving is voor ref.tst.vng.cloud.<br/><br/>Deze omgeving wordt door API Beheer beheerd.<br/><br/>
```

Comment:
Volgens mij wordt de documentatie vanuit github repositories gegenereerd

Reply:
Ja en nee :)

```
Gemma-Zaken Github (update) --> Jenkins (build Docker image) --> DockerHub (update)
Jenkins (trigger door Jenkins: build Docker image) --> Kubernetes (update, ref.tst.vng.cloud)
```

Dus, als Github bijgewerkt wordt, triggered een Jenkins job die de Docker image (van de documentatie) bouwt en opstuurt naar DockerHub. Jenkins triggered een andere job, die de image naar Kubernetes deployed, en daardoor zichtbaar is op ref.tst.vng.cloud.

Reply:
Even zien iof ik dan snap waar wat draait:
* Nu draait Kubernetes op het geleende deel van NLX maar tzt komt dat in Haven te draaien.

Waar draaien die Jenkins jobs? Ook op het deel bij NLX en straks bij Haven? Of nog weer ergens anders?

Reply:
> Nu draait Kubernetes op het geleende deel van NLX maar tzt komt dat in Haven te draaien.
✔️ 

> Waar draaien die Jenkins jobs? Ook op het deel bij NLX en straks bij Haven? Of nog weer ergens anders?
Nu nog bij NLX, dat kan straks bij Haven of bij een externe partij als Travis.

Reply:
Toppie, dank je!